src/Aquarius.Client.UnitTests/TimeSeries/Client/Helpers/ClientHelperTests.cs

```diff
@@ -70,6 +70,27 @@ public void SetAuthenticationToken_ExistingToken_NewTokenIsAdded()
             AssertThatTokenMatchesExpected(token2);
         }
 
+        [Test]
+        public void Login_IdentityProvider_SetsBearerTokenAsAccessToken()
+        {
+            var mockAccessToken = _fixture.Create<string>();
+            ClientHelper.Login(_rawClient, mockAccessToken);
+
+            _rawClient.BearerToken.ShouldBeEquivalentTo(mockAccessToken);
```

Comment:
I'm not sure if this is right, I would have though this would be:
`Bearer {mockAccessToken}` not just `mockAccessToken` as specified in https://datatracker.ietf.org/doc/html/rfc6750#page-5

I guess it could be prefixed at the time the header is being set, but I'm not seeing that in the code, could also be we are assuming that the token already includes the `Bearer ` prefix, but that seems like an assumption that should be checked.

Reply:
@MartinBenn-AI note, the `BearerToken` property on the `JsonServiceClient`, which the `SdkServiceClient` inherits from, is used by ServiceStack to set up the Authentication header automatically to save developers time. The value passed should be just the token, nothing extra, not "Bearer <access-token>".